### DISCUSSION TITLE
What is order of events for automated cold-start documentation ?

### DISCUSSION THREAD

**Discussion by seanlongnyc** [cloudposse/community#26] [Free Tier Support] [answered]
It looks like the order of events is
- Review Design
- Prepare AWS Org
- Init TF backend
- Deploy Account Baseline
- Deploy Accounts

But the "Deploy Account Baseline" (which precedes "Deploy Accounts") starts with the text "Now that all accounts have been deployed" & we when run we are seeing an error that the account-map does not exist which seems to make sense since we are working towards deploying accounts.
Based on this, should "Deploy Accounts" be before the baseline ?

![image](https://github.com/user-attachments/assets/afa223e8-0659-43d1-aefe-11acab28c3f5)

**Comment by milldr** [comment, marked as answer]
@seanlongnyc you're right, this is a mistake in the sidebar. "Deploy Accounts" should come before "Deploy Account Baseline". I'll create a task to fix this.

**Reply by seanlongnyc** [reply]
Awesome, Thanks for the quick response!

**Reply by GabisCampana** [reply]
@milldr https://linear.app/cloudposse/issue/DEV-2796/deploy-accounts-should-come-before-deploy-account-baseline

**Reply by milldr** [reply]
This is now fixed, as well as few additional improvements:
https://docs.cloudposse.com/layers/accounts/account-baseline/
